com.microsoft.mrtk.core/Editor/Inspectors/Utilities/MixedRealityInspectorUtility.cs

```diff
@@ -55,17 +55,14 @@ public static Color DefaultBackgroundColor
         public static readonly Texture2D LogoLightTheme = AssetDatabase.LoadAssetAtPath<Texture2D>(AssetDatabase.GUIDToAssetPath(LogoLightThemeGuid));
         public static readonly Texture2D LogoDarkTheme = AssetDatabase.LoadAssetAtPath<Texture2D>(AssetDatabase.GUIDToAssetPath(LogoDarkThemeGuid));
 
-        private const string CloneProfileHelpLabel = "Currently viewing a MRTK default profile. It is recommended to clone defaults and modify a custom profile.";
-        private const string CloneProfileHelpLockedLabel = "Clone this default profile to edit properties below";
-
-        private static readonly GUIStyle ProductNameStyle = new GUIStyle(EditorStyles.boldLabel)
+        public static readonly GUIStyle ProductNameStyle = new GUIStyle(EditorStyles.boldLabel)
         {
             fontSize = 26,
             alignment = TextAnchor.UpperCenter,
             fixedHeight = 32
         };
 
-        internal static bool IsMixedRealityToolkitLogoAssetPresent()
+        public static bool IsMixedRealityToolkitLogoAssetPresent()
```

Comment:
does this need to be public or could we use `InternalsVisibleTo`? It feels like a really specific method that others shouldn't need to worry about

Reply:
i can make that change

Reply:
done.